website/frontend/templates/docs/plugin-api.html

```diff
@@ -86,6 +121,10 @@ <h1 id="tagger_script_functions">Tagger Script Functions</h1>
       PLUGIN_NAME = &quot;Initials&quot;
       PLUGIN_AUTHOR = &quot;Lukas Lalinsky&quot;
       PLUGIN_DESCRIPTION = &quot;Provides tagger script function $initials(text).&quot;
+      PLUGIN_VERSION = '0.1'
+      PLUGIN_API_VERSIONS = ['2.0']
+      PLUGIN_LICENSE = "GPL-2.0"
```

Comment:
The above code uses `&quot;` instead of `"`. Is there a reason for not taking the same approach here?